pkg/k8s/utils/factory.go

```diff
@@ -248,6 +308,24 @@ func ResourceEventHandlerFactory(
 	return rehs
 }
 
+func getVerStructFrom(objMeta meta_v1.Object) (versioned.UUID, versioned.Object) {
```

Comment:
This function seems like it would be unnecessary if the map didn't expose the internal structure of how it stores objects.